mobilecoind-json/src/data_types.rs

```diff
@@ -39,8 +44,8 @@ impl From<&mc_mobilecoind_api::GetAccountKeyResponse> for JsonAccountKeyResponse
 #[derive(Deserialize, Default)]
 pub struct JsonMonitorRequest {
     pub account_key: JsonAccountKeyResponse,
-    pub first_subaddress: u64,
-    pub num_subaddresses: u64,
+    pub first_subaddress: String,
```

Comment:
These shouldn't be Strings. You're not gonna have that many subaddresses...

Reply:
Since they're u64 in the proto, it's probably best to treat them the same way as we treat the other u64s, as strings.

https://github.com/mobilecoinofficial/mobilecoin/blob/master/mobilecoind/api/proto/mobilecoind_api.proto#L177

Reply:
Comment intention understood - changing these to Strings breaks the downstream users. Returned back to u64.